### PR TITLE
Update base template styling

### PR DESCRIPTION
**Pull request by picklelo**
Don't use utf-8 encoding.

### CODE REVIEW

**Comment by masenf** [comment]
why is the encoding fix necessary? shouldn't we be reading these files as text?

**Comment by picklelo** [comment]
Undid the utf-8 encoding. This PR is now for just updating the base template styling and adding the page kwargs to the template.

**Review by masenf — COMMENTED** [review]
sidebar doesnt show up at all on mobile now, not clear how to display it, am i missing something?

**Comment by picklelo** [comment]
@masenf That is the intention - I moved the links to the top menu bar for mobile.

**Comment by masenf** [comment]
Oh alright i see that now.. second question, why does the content always scroll off the page?

File: reflex/.templates/apps/demo/code/sidebar.py [hunk 1]
```diff
@@ -169,6 +169,7 @@ def sidebar() -> rx.Component:
             sidebar_footer(),
             height="100dvh",
         ),
+        display=["none", "none", "flex"],
```
Comment (masenf) [inline reflex/.templates/apps/demo/code/sidebar.py:172]:
```suggestion
        display=["none", "none", "block"],
```

**Review by masenf — CHANGES_REQUESTED** [review]

**Review by masenf — DISMISSED** [review]

**Review by masenf — APPROVED** [review]

**Pull request merged**